### PR TITLE
Enhancement: add web console for operations

### LINKED ISSUE #267: Feature request: web console for operations

### LINKED ISSUE #269: Provide dashboard

### PR DESCRIPTION
**Pull request by ansjsun**
**What this PR does / why we need it**:

- Add console component for operation.

**Which issue this PR fixes**: 
- fixes #267 
- fixes #269 

**Special notes for your reviewer**:
None.

**Release note**:

```release-note
Enhancement:
- Add web console for operations.
```

### CODE REVIEW

**Comment by hooklee2000** [comment]
How about commit the source code of html and javascript? After webpack,we can'nt understand and modify the web console.

File: console/cutil/handler.go [hunk 1]
```diff
@@ -0,0 +1,839 @@
...
+
+package cutil
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/chubaofs/chubaofs/proto"
+	client2 "github.com/chubaofs/chubaofs/sdk/graphql/client"
```
Comment (awzhgw) [inline console/cutil/handler.go:22]: use graphqlClient replace client 2

File: console/cutil/handler.go [hunk 2]
```diff
@@ -0,0 +1,839 @@
...
+	}
+
+	if r.Body == nil {
+		writeResponse(w, http.StatusBadRequest, nil, errors.New("request must include a query"))
+		return
+	}
+
+	ctx := r.Context()
+
+	if r.RequestURI != "/login" {
```
Comment (awzhgw) [inline console/cutil/handler.go:188]: use const variable

File: cmd/cfg/console.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "logDir": "/cfs/log/",
+  "logLevel": "debug",
+  "listen": "80",
+  "master_instance": "11.5.115.39:9066",
+  "masterAddr": [
+    "192.168.0.11:17010",
+    "192.168.0.12:17010",
+    "192.168.0.13:17010"
+  ],
+  "monitor_addr": "http://cfsthanos-yf01.cfsmon.svc.zyx02.n.jd.local",
```
Comment (awzhgw) [inline cmd/cfg/console.json:12]: master_instance and monitor_addr

File: console/cutil/handler.go (hunk 1 above)
Comment (mervinkid) [reply console/cutil/handler.go:22]: It is recommended to name it like `gqlclient`.

**Comment by ansjsun** [comment]
@hooklee2000 The current submission is webpack. In the future, we may create a new respository to store page source code.

**Pull request merged**